tests/worker/test_task_runner_worker.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from airflow.worker import task_runner_worker
+import unittest
+
+
+class TestTaskRunnerWorker(unittest.TestCase):
+    def setUp(self) -> None:
+        app = task_runner_worker.create_app()
+        self.app = app.test_client()
+
+    def test_hello(self):
+        resp = self.app.get("health", query_string={'name': 'daniel'})
+        self.assertEqual(resp.status_code, 200)
+
+    def test_passing_task(self):
+        dag_id = 'example_python_operator'
+        task_id = 'print_the_context'
+        import datetime
+        start_date = datetime.datetime(year=2019, day=1, month=1)
+        exec_date = int(datetime.datetime.timestamp(start_date))
```

Comment:
The following should also work:

```suggestion
        exec_date = start_date.timestamp()
```


```python
In [1]: import datetime
In [2]: test_date = datetime.datetime(year=2019, day=1, month=1)
In [3]: test_date.timestamp
Out[3]: <function datetime.timestamp>
In [4]: test_date.timestamp()
Out[4]: 1546300800.0
```